src/plugins/Webcam/index.js

```diff
@@ -168,7 +169,9 @@ module.exports = class Webcam extends Plugin {
       })
       return this.getVideo()
     }).then((file) => {
-      return this.uppy.addFile(file)
+      this.uppy.addFile(file)
+      const dashboard = this.uppy.getPlugin('Dashboard')
+      if (dashboard) dashboard.hideAllPanels()
```

Comment:
why is this done here? i feel like the webcam plugin shouldn't know it's in a panel

Reply:
Yeah, ideally not, but we have to figure out a way to close tab after picture is taken. Previously we used “when file is added, close panel” login in Dashboard, but that doesn’t work with selecting files via checkboxes.